config/default.yaml

```diff
@@ -0,0 +1,6 @@
+defaults:
+  - dataset: nerf_synthetic
```

Comment:
By default, `nerf_synthetic` dataset is not placed under `data` dir. Instead `test_metalball` exists from the beginning, right? If so, it might be better to switch the default value to `test` because it allows to run `neddf/scripts/run.py` with default option from the beginning.

Reply:
I got it.
I think `smoke bunny` is better to sample. This is an opportunity to add a dataset and make it the default
I'll make other PR.